lib/commands/geolocation.js

```diff
@@ -38,11 +49,18 @@ commands.mobileSetSimulatedLocation = async function mobileGetSimulatedLocation
  * ! Do not forget to reset the simulated geolocation value after your automated test is finished.
  * ! If the value is not reset explcitly then the simulated one will remain until the next device restart.
  *
+ * @returns {Promise<void>}
```

Comment:
In the case where we expect `void`, using `return await` below may not adhere to that contract--since TS doesn't know anything about what `this.proxyCommand(anything)` does and just agrees with what we tell it.